shared/Cargo.toml

```diff
@@ -8,6 +8,7 @@ description = "EDHOC crypto library constants crate"
 repository.workspace = true
 # It's implied, but still better for consistency to have it explicit.
 readme.workspace = true
+keywords.workspace = true
```

Comment:
not sure if categories fit in here too -- after all, this is not the primary-to-discover crate, but if keywords are good to have here, so will be categories.